source/database/mssql/scripts/dbscripts/PSP_PIMS_S97_00/Build/051_DML_PIMS_ROLE.sql

```diff
@@ -0,0 +1,20 @@
+DELETE FROM PIMS_ROLE
+GO
+
+INSERT INTO PIMS_ROLE (ROLE_UID, NAME, DESCRIPTION, IS_PUBLIC, IS_DISABLED, SORT_ORDER, APP_CREATE_TIMESTAMP, APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_LAST_UPDATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_DIRECTORY)
+VALUES
+  (NEWID(), N'System Administrator', N'System Administrator of the PIMS solution.',                  CONVERT([bit],(0)), CONVERT([bit],(0)), 0, CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data'),
+  (NEWID(), N'Undetermined', N'The user has an undetermined role with their organization.',          CONVERT([bit],(0)), CONVERT([bit],(1)), 0, CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data'),
+  (NEWID(), N'Acquisition functional',   N'Access to create, read, update Acquisition files.',       CONVERT([bit],(1)), CONVERT([bit],(0)), 0, CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data'),
+  (NEWID(), N'Acquisition read-only',    N'Access to read Acquisition files.',                       CONVERT([bit],(1)), CONVERT([bit],(0)), 0, CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data'),
+  (NEWID(), N'Lease/License functional', N'Access to create, read, update Leases/Licenses files.',   CONVERT([bit],(1)), CONVERT([bit],(0)), 0, CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data'),
+  (NEWID(), N'Lease/License read-only',  N'Access to read lease/license.',                           CONVERT([bit],(1)), CONVERT([bit],(0)), 0, CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data'),
```

Comment:
@dfilteau please coordinate with @devinleighsmith regarding changes to the lease license role name.

See his PR here: https://github.com/bcgov/PSP/pull/4545/files

We would need to keep those changes in this and all future DB migrations

Reply:
I understand you don't touch those. But that is why I'm bringing it to your attention so you can coordinate with Devin and put those changes in 97. Otherwise we will loose it when this PR gets merged